examples/example.hpp

```diff
@@ -86,6 +85,9 @@ class texture
         case RS2_FORMAT_RGB8:
             glTexImage2D(GL_TEXTURE_2D, 0, GL_RGB, width, height, 0, GL_RGB, GL_UNSIGNED_BYTE, frame.get_data());
             break;
+        case RS2_FORMAT_Y8:
```

Comment:
Y8 rendering is available in rendering.h/texture_buffer. you could reuse that class